jhove-core/src/main/java/edu/harvard/hul/ois/jhove/RAFInputStream.java

```diff
@@ -19,106 +21,117 @@
 public class RAFInputStream extends InputStream {
 
-    /* The file on which the stream is based */
+    /** Default file buffer size */
+    private static int DEFAULT_BUFFER_SIZE = 65536;
+
+    /** The file on which the stream is based */
     private RandomAccessFile _raf;
     
-    /* Size of fileBuf */
+    /** Size of fileBuf */
     private int fileBufSize;
     
-    /* Buffer for reading from the file */
+    /** Buffer for reading from the file */
     private byte[] fileBuf;
     
-    /* Offset for reading next byte from filebuf */
+    /** Offset for reading next byte from fileBuf */
     private int fileBufPos;
     
-    /* Number of valid bytes in fileBuf */
+    /** Number of valid bytes in fileBuf */
     private int fileBufBytes;
     
-    /* Position in file for next read from RandomAccessFile */
+    /** Position in file for next read from RandomAccessFile */
     private long fileOffset;
     
-    /* EOF flag */
-    boolean eof;
+    /** End-of-file flag */
+    private boolean eof;
     
     /**
-     *  Constructor with default buffer size.
-     *  The stream starts at the current position of the
-     *  RandomAccessFile.
-     * 
-     *  @param   raf   The file on which the
-     *                 stream is to be based.
+     * Constructor with default buffer size.
+     *
+     * The stream starts at the current position of the RandomAccessFile.
+     *
+     * @param   raf   the file on which the stream is to be based.
      */
-    public RAFInputStream(RandomAccessFile raf) {
+    public RAFInputStream(RandomAccessFile raf)
+    {
         super();
         _raf = raf;
-        fileBufSize = 65536;  // pick a size, any size
-        init ();
+        fileBufSize = DEFAULT_BUFFER_SIZE;
+        init();
     }
 
-
     /**
-     *  Constructor with buffer size.
-     *  The stream starts at the current position of the
-     *  RandomAccessFile.
-     * 
-     *  @param   raf         The file on which the
-     *                       stream is to be based.
-     * 
-     *  @param   bufferSize  The buffer size to be used.
-     *                       If less than or equal to 0, the
-     *                       default buffer size is used.
+     * Constructor with buffer size.
+     *
+     * The stream starts at the current position of the RandomAccessFile.
+     *
+     * @param   raf         the file on which the stream is to be based.
+     *
+     * @param   bufferSize  the buffer size to be used. If less than or
+     *                      equal to 0, the default buffer size is used.
      */
-    public RAFInputStream(RandomAccessFile raf, int bufferSize) {
+    public RAFInputStream(RandomAccessFile raf, int bufferSize)
+    {
         super();
         _raf = raf;
-        fileBufSize = (bufferSize <= 0 ? 65536 : bufferSize);
-        init ();
+        fileBufSize = (bufferSize <= 0 ? DEFAULT_BUFFER_SIZE : bufferSize);
+        init();
     }
 
-    private void init ()
+    private void init()
```

Comment:
Super pernickety and I have no good answer but the constructor above is corrected with the open bracket alongside the declaration (correct IMO) but for init just the spacing is corrected. Sorry take that back, mis-read the above and spent too much time recently working through JHOVE's inconsistent spacing / formatting issues.